modules/user/src/main/User.scala

```diff
@@ -310,7 +311,8 @@ object User:
         plan = r.getO[Plan](plan) | Plan.empty,
         totpSecret = r.getO[TotpSecret](totpSecret),
         flair = r.getO[Flair](flair).filter(FlairApi.exists),
-        marks = r.getO[UserMarks](marks) | UserMarks.empty
+        marks = r.getO[UserMarks](marks) | UserMarks.empty,
+        hasEmail = r.getO[EmailAddress](email).isDefined
```

Comment:
We don't even need to parse the email in theory, just check that the field exists, but not found a method to do that.

Reply:
`r.contains(email)` should work?